R/chooseFromList.R

```diff
@@ -46,7 +46,7 @@ chooseFromList <- function(theList, type = "items", userinfo = NULL, addAllPatte
   }
   if (multiple) {
     groupsids <- NULL
-    if (exists("groups")) {
+    if (length(rawNames) > 0) {
```

Comment:
If the condition in line 44 changes this would also need to change, so I'd suggest
```suggestion
    if (!is.null(groups)) {
```
and adding `groups <- NULL` between line 43 and 44 (right before the `if` block that may set `groups`).

Reply:
done